examples/react/css/src/common.css

```diff
@@ -0,0 +1,61 @@
+/* Motion reduction */
+@media (prefers-reduced-motion: reduce) {
+  [class^='ais-'] {
+    transition: none !important;
+  }
+}
+
+/* Focus outline */
+a[class^='ais-']:focus-visible,
+button[class^='ais-']:focus-visible,
+input[class^='ais-']:focus-visible {
+  outline: 2px solid
+    rgba(var(--ais-primary-color-rgb), var(--ais-primary-color-alpha));
+  outline-offset: 2px;
+}
```

Comment:
I put it there for now, inspired by what we do in Satellite, we have a global CSS class `focusable` (`fosauble-visible`, etc...) to keep our focus styles consistent across components.

Reply:
Sounds great! Let's just make sure it doesn't clash with the focus style for the search box.

Reply:
`display: none` or `appearance: none;` are both working on my side

Reply:
It should only be present when you use the keyboard to navigate on the page